cli/lib/kontena/stacks_cache.rb

```diff
@@ -20,7 +20,7 @@ def read
       end
 
       def load
-        YAML.safe_load(read)
+        ::YAML.safe_load(read, [], [], true, path)
```

Comment:
no coverage

Reply:
Is this `load` method even called somewhere? All the users of `Kontena::StacksCache` that I can find use `pull`, and that's a separate code path with it's own `::YAML.safe_load`. I suspect this is actually dead code?